block/manager.go

```diff
@@ -243,6 +252,18 @@ func (m *Manager) SetDALC(dalc *da.DAClient) {
 	m.dalc = dalc
 }
 
+// isProposer returns whether or not the manager is a proposer
+func isProposer(genesis *cmtypes.GenesisDoc, signerPrivKey crypto.PrivKey) (bool, error) {
+	if len(genesis.Validators) == 0 {
+		return false, errors.New("no validators found in genesis")
```

Comment:
I'd make this a global variable so that you can reference it directly in the tests.

Reply:
Updated